sandbox_storage/scripts/populate.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2021 Universität Tübingen, DKFZ and EMBL
+# for the German Human Genome-Phenome Archive (GHGA)
+
+# Can't populate S3 yet
+# So, just take example files and hard links right now
+
+"""
+Provides a script function to populate a database
+"""
+
+from datetime import datetime
+from os import listdir
+from os.path import isfile, join, getsize, getctime
+import hashlib
+from sqlalchemy.exc import IntegrityError
+
+
+from ..database import get_session
+from ..config import get_settings
+from ..models import DrsObject
+
+
+settings = get_settings()
+DIR_PATH = settings.example_files_path
```

Comment:
```suggestion
HERE = Path(__file__).parent.resolve()
DIR_PATH = HERE.parent.resolve() / "examples"
```
We can directly hardcode the location of examples here, relative to the script.